content/participants/ttjc@harshdigwani/index.md

```diff
@@ -32,12 +46,109 @@ It would be great if someone help me out in improving my backend skills.
+
+_stack_ : Reactjs, 
+        Nodejs, 
+        ExpressJS, 
+        Socket.io for realtime communication, 
+        HTML, CSS and Flexbox for Responsive Design, 
+        Netlify for deployment.
+
+_hosted link_ : https://1chat.netlify.app/
+
+_github link frontend_ : https://github.com/harshdigwani/onechatfrontend
+
+_github link backend_ : https://github.com/harshdigwani/onechatbackend
+
+
+### Bookmarkit!
+
+_description_ : Bookmarkit is a progressive web application which save users bookmark and also use to share text between any devices directly.
+
+_stack_ : Reactjs, 
+        Nodejs, 
+        Firebase firestore for Database, 
+        Firebase Auth for Authentication,
+        Firebase storage for storing files on cloud, 
+        Socket.io for realtime communication, 
+        HTML, CSS and Flexbox for Responsive Design, 
+        Netlify for deployment.
+
+_hosted link_ :  http://bookmarkitapp.netlify.app/
+
+_github link_ :  https://github.com/harshdigwani/bookmarkit
 
 
-My projects:
+### Profile
 
-#### Gokart: An Ecommerce WebApp
+_description_ : Profile is Blog and Project sharing along with E Resume which sorts the process of applying and filling repeated details
+
+_stack_ : Reactjs, 
+        Nodejs, 
+        ExpressJS,
+        Authentication,
+        Jest for Unit and Integration Testing, 
+        Redux for state management, 
+        HTML, CSS and Flexbox for Responsive Design, 
+        Netlify for deployment.
+
+_hosted link_ :  http://profileshare.netlify.app/
+
+_github link frontend_ : https://github.com/harshdigwani/profilefrontend
+
+_github link backend_ : https://github.com/harshdigwani/Profilebackend
+
+
+### Emophony : Emotion Recognition based Music Recommendation
+
+_description_ :  It is a web application which use face emotion recognisation using Convolutional Neural Network(CNN) based deep learning model and recommand some list of songs to user for emotion detected.
+
+_stack_ : Python 3+,
+        Flask Framework for developing API for Deep Learning model,
+        Javascript for making Single Page Application,
+        HTML, CSS and Flexbox for Responsive Design.
+
+_hosted link_ : https://emophony.netlify.app/
+
+_github link_ : https://github.com/harshdigwani/Emophony
+
+
+### Portfolio v2 
+
+_description_ :  Updating my protfolio to second version removing the use of bootstrap an now it is build without any CSS Library only using Pure CSS.
+
+Currently working...
+
+_stack_ : HTML, CSS and Flexbox and Media Queries.
+
+_hosted link_ : https://harshdigwani.github.io/ <br/>
+_Old Version_   https://harshdigwani.github.io/public/v1/index.html<br/>
+_github link_ : https://github.com/harshdigwani/harshdigwani.github.io
+
+
+
+### Gokart: An Ecommerce WebApp
 
 _description_ Gokart an E-Commerce application which solves the problem of finding the best deals on electronic products near you using user's location.
 
```

Comment:
Please remove old projects, after that you can raise **PR** for the finalist

Reply:
Done sir!